Makefile

```diff
@@ -16,4 +16,4 @@ vet: ## Run go vet against code
 
 .PHONY: tests
 tests: ## Run all tests and requires a running rabbitmq-server
-	env AMQP_URL=amqp://guest:guest@127.0.0.1:5672/ go test -race -v -tags integration
+	env AMQP_URL=amqp://guest:guest@127.0.0.1:5672/ go test -cpu 1,2 -race -v -tags integration
```

Comment:
Why are the tests actually limited to run only on core 1 & 2?

Reply:
I have no idea! Maybe to reduce contention with RabbitMQ? It's probably not necessary.

Reply:
There's no additional context in the initial commit that set the `-cpu` flag 87488be91988d8e38f0d9f5c43f33bd5d2174e37. I agree it's probably not necessary.

Reply:
AHA! I bet this is why: https://splice.com/blog/lesser-known-features-go-test/

> By default, unless GOMAXPROCS is set via an environment variable or explicity in the code, the tests are run with GOMAXPROCS=1. One can type GOMAXPROCS=2 go test to test with 2 CPUs, and then type GOMAXPROCS=4 go test to test with 4, but there is a better way: go test -cpu=1,2,4 will run the tests three times, with 1, 2 and 4 as GOMAXPROCS values.